.gitignore

```diff
@@ -0,0 +1,7 @@
+index-v-01.html
+resize.html
+resize2.html
+style-old.css
```

Comment:
Please remove files from the repo and from the .gitignore that should not be in the repo.